src/auth/support.js

```diff
@@ -400,16 +399,8 @@ export async function getAuthInfo(context, info) {
         // }
 
         const { attributes: { config, orgConfig } } = context;
-        let siteApiKeyId = config?.access?.admin?.apiKeyId || [];
-        if (!Array.isArray(siteApiKeyId)) {
-          siteApiKeyId = [siteApiKeyId];
-        }
-        let orgApiKeyId = orgConfig?.access?.admin?.apiKeyId || [];
-        if (!Array.isArray(orgApiKeyId)) {
-          orgApiKeyId = [orgApiKeyId];
-        }
-        if (siteApiKeyId.indexOf(profile.jti) < 0 && orgApiKeyId.indexOf(profile.jti) < 0) {
-          const apiKeyId = [...siteApiKeyId, ...orgApiKeyId];
+        const apiKeyId = config?.access?.admin?.apiKeyId || orgConfig?.access?.admin?.apiKeyId;
+        if (apiKeyId.indexOf(profile.jti) < 0) {
```

Comment:
use either `site` config `apiKeyId` array, or fallback to `org`, if there's no site config